docs/content/getting-started/quickstart.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Dagster Quickstart | Dagster Docs
+description: Run dagster for the first time
+---
+
+# Dagster Quickstart
+
+Welcome to Dagster! In this guide, you'll get up-and-running with the [Dagster Quickstart](https://github.com/dagster-io/dagster-quickstart) project. This project demonstrates the power of Dagster, and provides a starting point to allow further exploration of Dagster's features.
```

Comment:
Should we add a **NOTE** saying, "If you are looking to scaffold a new project instead, see ..."

Reply:
I've added it for now.